core/templates/pages/learner-dashboard-page/new-learner-dashboard/community-lessons-tab.component.html

```diff
@@ -0,0 +1,654 @@
+      </div>
+    </div>
+  </div>
+</div>
+<div class="subscriptions-section">
+  <p class="lessons-title" [innerHTML]="'I18N_LEARNER_DASHBOARD_SUBSCRIPTIONS_SECTION' | translate"></p>
+  <div *ngIf="subscriptionsList.length === 0">
+    <p [innerHTML]="'I18N_LEARNER_DASHBOARD_EMPTY_SUBSCRIPTIONS' | translate" class="empty-section"></p>
+    <a class="btn oppia-dashboard-intro-button oppia-transition-200 oppia-learner-dashboard-intro-button-link"
+      [innerHTML]="'I18N_ACTION_BROWSE_LESSONS' | translate" href="/community-library">
+    </a>
+  </div>
+  <div *ngIf="subscriptionsList.length !== 0" class="oppia-subscriptions-container">
+    <mat-card class="oppia-subscription-card oppia-subscription-container-list"
+      *ngFor="let subscription of subscriptionsList">
+      <a class="oppia-subscription-profile-link" href="/profile/{{ subscription.username }}" target="_blank" rel="noopener">
+        <img [src]="decodePngURIData(subscription.pictureDataUrl)"
+             class="oppia-subscription-card-profile-picture rounded-circle">
+        <span class="oppia-subscription-card-summary text-dark">
+          <div class="oppia-subscription-name">
+            <strong container="body"
+                    triggers="showUsernamePopover(subscription.username)" ngbPopover="subscription.username">
+              {{ subscription.username| truncate:10 }}
+            </strong>
+          </div>
+          <div class="oppia-impact-text">
+            <span>Impact: </span>
+            <span>{{ subscription.impact || 0 }}</span>
+          </div>
+        </span>
+      </a>
+    </mat-card>
+    <div class="oppia-toggle-button-section" *ngIf="subscriptionsList.length > 3">
+      <button class="btn oppia-toggle-btn"
+              (click)="handleShowMore('subscription')"
+              *ngIf="!showMore['subscription'] && !checkMobileView()">
+              Show More
+      </button>
+      <button class="btn oppia-toggle-btn"
+              (click)="handleShowMore('subscription')"
+              *ngIf="showMore['subscription'] && !checkMobileView()">
+              Show Less
+      </button>
+    </div>
+  </div>
+</div>
+<style>
+  .community-lessons-section,
+  .playlist-section,
+  .subscriptions-section {
+    background-color: #fff;
+    border-radius: 4px;
+    box-shadow: 0 0 4px #0000001a;
+    margin: auto;
+    margin-bottom: 25px;
+    min-height: 215px;
+    opacity: 1;
+    padding: 0 0 30px 0;
+    width: 726px;
+  }
+  .summary-container {
+    margin: auto;
+    width: 656px;
+  }
+  .oppia-dashboard-intro-button {
+    left: 40px;
+    position: relative;
+  }
+  .pagination {
+    bottom: 10px;
+    display: block;
+    list-style: none;
+    padding-left: 0;
+    position: relative;
+    text-align: end;
+    width: 250px;
+  }
+  .pagination-bottom {
+    bottom: 10px;
+    display: block;
+    list-style: none;
+    padding-top: 25px;
+    position: relative;
+    right: 41px;
+    text-align: end;
+  }
+  .dot {
```

Comment:
Can this be more descriptive? Where is it used?

Reply:
Done. The pagination number is displayed on this dot/circle.